sentry-android-core/build.gradle.kts

```diff
@@ -64,6 +66,27 @@ android {
     }
 }
 
+tasks {
+    val androidJavadocs by creating(Javadoc::class) {
+        source = android.sourceSets["main"].java.getSourceFiles()
+    }
+
+    val androidJavadocsJar by creating(Jar::class) {
+        archiveClassifier.set("javadoc")
+        from(androidJavadocs.destinationDir)
```

Comment:
this one looks different than https://github.com/vanniktech/gradle-maven-publish-plugin/blob/master/src/main/kotlin/com/vanniktech/maven/publish/tasks/AndroidJavadocsJar.kt#L18-L20